SU2_CFD/include/variable_structure.hpp

```diff
@@ -3638,6 +3649,10 @@ class CTurbVariable : public CVariable {
  */
 
 class CTurbSAVariable : public CTurbVariable {
+
+private:
+  su2double gamma_BC; /*!< \brief Value of the intermittency for the BC trans. model. */
```

Comment:
I am so busy with my work and phd studies, and thus cannot find time to work on coupling my model with SST. So, we may leave it as is now, if thats fine.